packages/frontend/components/atomic/CommunityPicker/CommunityPicker.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Story } from "@storybook/react";
+import CommunityPicker from ".";
+type Props = React.ComponentProps<typeof CommunityPicker>;
+
+export default {
```

Comment:
nit: Since the community picker uses the Dropdown component, let's remove the CommunityPicker story.